src/routes/index.ts

```diff
@@ -286,7 +287,7 @@ router.route(config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL)
     navigation.hasOverseasEntity
   )
   .get(confirmOverseasEntityDetails.get)
-  .post(confirmOverseasEntityDetails.post);
+  .post(companyAuthentication, confirmOverseasEntityDetails.post);
```

Comment:
We will need to add the companyAuthentication middleware on all subsequent pages starting from this one.
For the same reason the login authentication is there on each page aa well.

Reply:
can add it to all subsequent pages, but with the navigation checks a user cannot skip the Auth code page, same with login it only needs to be added to the first page , but it does make it clear to understand on what is needed, so will add it to all subsequent pages

Reply:
Done now.